### DISCUSSION TITLE
Enable cheat category commands on the PTR.

### DISCUSSION THREAD

**Discussion by heyitsbench** [chromiecraft/chromiecraft#3963] [Suggestions]
Maybe not every one of them, but something like `.cheat cooldown` can be helpful for testing spells with high cooldowns.

**Comment by ohsiros** [comment]
You can reset cooldowns with .cooldown

**Comment by heyitsbench** [comment]
Yea this hasn't been resolved.

**Reply by Amrx2** [reply]
Should be after the next restart. If not can you let me know, thanks.

**Reply by heyitsbench** [reply]
Only the `.cheat god`  subcommand is available as of April 2nd.
